INSTALL

```diff
@@ -615,7 +615,7 @@ The directories set up by Configure fall into three broad categories.
 
 =item Directories for the perl distribution
```

Comment:
INSTALL will need a note that as of 7.0.0 the prefix will be `.../lib/perl` instead of `.../lib/perl5` (which is likely to break some code that analyses already installed modules)

As a side note, will perl-7.x.x be able to load modules from perl-5.x.x when `inc_version_list` not equals to `none`.
If so, do we need a new parameter to `Configure` to define that location?